pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -1190,6 +1193,28 @@ type CNIConfig struct {
 	CNIUrls []string `yaml:"urls,omitempty"`
 }
 
+// ExternalCloudProviderConfig contains external cloud provider configuration.
+type ExternalCloudProviderConfig struct {
+	//   description: |
+	//     Enable external cloud provider.
+	//   values:
+	//     - true
+	//     - yes
+	//     - false
+	//     - no
+	ExternalEnabled bool `yaml:"enabled,omitempty"`
+	//   description: |
+	//     A list of urls that point to additional manifests for an external cloud provider.
+	//     These will get automatically deployed as part of the bootstrap.
+	//   examples:
+	//     - value: >
+	//        []string{
+	//         "https://raw.githubusercontent.com/kubernetes/cloud-provider-aws/v1.20.0-alpha.0/manifests/rbac.yaml",
+	//         "https://raw.githubusercontent.com/kubernetes/cloud-provider-aws/v1.20.0-alpha.0/manifests/aws-cloud-controller-manager-daemonset.yaml",
+	//        }
+	Manifests []string `yaml:"manifests,omitempty"`
```

Comment:
for consistency sake, should we name variable `ExtraManifests` and interface method `Manifests`?

Reply:
Should we rename the YAML config field too? How "extra" are those manifests? My understanding is that required more often than not, and those are not "extra". And then I would keep the struct field name the same as the config field name.

Reply:
sorry, `ExternalCloudProviderManfiests` or whatever, as we usually prefix field names with struct name, while interface functions are short names

Reply:
Renamed field to `ExternalManifests`. Method name is `ManifestURLs` for consistency with existing `ExtraManifestURLs`.